Formula/asimov.rb

```diff
@@ -0,0 +1,44 @@
+class Asimov < Formula
+  desc "Automatically exclude development dependencies from Time Machine backups"
+  homepage "https://github.com/stevegrunwell/asimov"
+  url "https://github.com/stevegrunwell/asimov/archive/v0.2.0.tar.gz"
+  sha256 "2efb456975af066a17f928787062522de06c14eb322b2d133a8bc3a764cc5376"
+  head "https://github.com/stevegrunwell/asimov.git", :branch => "develop"
+
+  # No bottling necessary
```

Comment:
No need to explain the homebrew DSL does. We have https://docs.brew.sh/Formula-Cookbook for that already. Please remove all comments that explain parts of homebrew.

Reply:
Comments have been removed save for one, meant to explain at-a-glance that the .plist file schedules Asimov to run daily.